src/web_socket.toit

```diff
@@ -40,8 +41,10 @@ class WebSocket:
   pending_ /ByteArray := #[]
   current_writer_ /WebSocketWriter? := null
   current_reader_ /WebSocketReader? := null
+  client_ /bool
```

Comment:
I think `client` is ok as argument for the constructor, but as a field I would prefer a boolean name.
```suggestion
  is_client_ /bool
```